.github/workflows/release.yml

```diff
@@ -279,11 +279,23 @@ jobs:
           # Add additional line before the content
           RELEASE_NOTES="✅ New *${{ github.event.repository.name }}* Release [${{ needs.version-tag.outputs.version }}](${RELEASE_URL}) $(date)\n\n${RELEASE_NOTES}\n\nCommit Details:\n${new_fix_notes}"
 
```

Comment:
One suggestion for improvement could be to consolidate the multiple `sed` operations into a single command to improve efficiency. For example, combining the `sed` commands that remove '##', extra HTML, replace markdown links, and replace usernames into one `sed` command with multiple expressions would make the code cleaner and potentially faster.